src/mono/sample/wasm/browser-es6/runtime.js

```diff
@@ -0,0 +1,21 @@
+import createRuntime from './dotnet.js'
+
+export const { MONO, INTERNAL, BINDING, Module } = await createRuntime(({ MONO, INTERNAL, BINDING, Module }) => ({
```

Comment:
This is a weird signature - do I understand it right and it's like this?
```createRuntime (callback : Function) : Promise<{MONO,INTERNAL,BINDING,Module}>```
and then callback is ```callback (namespaces : {MONO,INTERNAL,BINDING,Module}) : ConfigObject```? How are the namespaces visible to the config getter if the runtime hasn't loaded yet? What will happen if it calls methods on them?